scalableminds/webknossos
# Fix context menu positioning

The positioning of the context menu is faulty because of #5658.
This is a quick bug fix that simply adds the css property again, which was removed in #5658, which introduced the bug.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a tracing and open the context menu. It should appear directly next to the mouse pointer and not at the top of the page with full page with.

### Issues:
- fixes a bug

------
